spec/ddtrace/contrib/delayed_job/active_record_setup.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.configure do |c|
```

Comment:
This is an improvement from a syntax perspective, but it now auto-runs around every example in the test suite. Normally we only run one integration at a time in our CI, but we shouldn't operate on that kind of assumption, as it may not always be true. This should only run around the examples it needs to, when the example explicitly configures itself to do so. (Not by virtue of `require`.)